test/test_jit_bindings.py

```diff
@@ -0,0 +1,194 @@
+from test_jit import JitTestCase
+from torch.testing._internal.common_utils import run_tests
+
+import torch
+
+
+class TestBindings(JitTestCase):
```

Comment:
Will this test actually get run? Do I need to add it to some other file or lists of tests?

Reply:
You need to add it to this list: https://github.com/pytorch/pytorch/blob/7a178a8a523d4653a3a2fa10c573b71e7fab1b9a/test/run_test.py#L20